recipes/arcsv/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: arcsv
+  version: 1.0.2
+
+source:
+  url: https://raw.githubusercontent.com/xuxif/ArcSV/main/archive/arcsv.tar.gz
+  sha256: 118b898af3fcb0618ccbd924728ee2ac520a23d75ee042dfdc86dfe1bf3276af
+build:
+  noarch: generic
+  number: 0
+  
+
+requirements:
+  run:
+    # List dependencies here. e.g.:
+    # ... any other dependencies 
+    - samtools
+    - bcftools
+    - bedtools
+test:
+  commands:
+    - ArcSV detect 
+
+about:
+  home: https://github.com/xuxif/ArcSV
+  license: GPL-3.0
```

Comment:
```suggestion
  license: GPL-3.0
  license_file: LICENSE
```

Reply:
Please add and fill in the license_file field with the license packaged from source.

Reply:
@mencian  

Hello,

I've updated the meta.yaml and added the license.txt file as per the suggestions. Could you please review the changes in my pull request? I appreciate your assistance.

Thank you (3Q).